test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -996,6 +1004,199 @@ TEST_F(ClientLbEnd2endTest, RoundRobinSingleReconnect) {
   WaitForServer(stub, 0, DEBUG_LOCATION);
 }
 
+// A minimal forwarding class to avoid implementing a standalone test LB.
+class ForwardingLoadBalancingPolicy : public grpc_core::LoadBalancingPolicy {
```

Comment:
This forwarding class needs to handle one more thing, which is the re-resolution callback.  You can model this off of the way that the grpclb policy delegates it to its nested round_robin policy.  Basically, you need to create your own re-resolution callback and pass it to the delegate policy, and when the child policy invokes your callback, you can in turn invoke the callback that the client_channel has given the forwarding class.

For an example, see [`GrpcLb::OnRoundRobinRequestReresolutionLocked()`](https://github.com/grpc/grpc/blob/master/src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc#L1748).